ee/clickhouse/sql/person.py

```diff
@@ -89,6 +89,33 @@
 GROUP BY distinct_id
 """
 
+# Using any() to make GROUP BY cheaper
+# See https://clickhouse.tech/docs/en/sql-reference/aggregate-functions/reference/any/
+GET_LATEST_PERSON_WITH_DISTINCT_IDS_SQL = """
```

Comment:
Note: This query is 100% this view specific and _should not_ be used anywhere else.

Problem is that GET_LATEST_PERSON_SQL needs to die ASAP - it doesn't allow for querying materialized columns. The proper replacement is `ClickhousePersonQuery` which can account for this.

However since this endpoint is doing the _wrong thing_ :tm: fundamentally around filtering properties this works for now.

Given that this kind of is one query specific hack, let's:
1. Refactor out the query from within the endpoint (there's an existing open thread about this)
2. Move the const there to signify this is that-usage specific
3. Add a comment to that effect.